02_configure_host.sh

```diff
@@ -342,3 +342,16 @@ fi
 # stage
 sudo virsh net-list | grep ${PROVISIONING_NETWORK_NAME} || sudo virsh net-start ${PROVISIONING_NETWORK_NAME}
 sudo virsh net-list | grep ${BAREMETAL_NETWORK_NAME} || sudo virsh net-start ${BAREMETAL_NETWORK_NAME}
+
+
+# Setup a single nfs export for image registry
+if [ "${PERSISTENT_IMAGEREG}" == true ] ; then
+    sudo rm -rf /opt/dev-scripts/nfsshare
+    sudo mkdir -p /opt/dev-scripts/nfsshare/1
+    echo "/opt/dev-scripts/nfsshare ${EXTERNAL_SUBNET_V6:-$EXTERNAL_SUBNET_V4}(rw,sync,no_subtree_check)" | sudo dd of=/etc/exports.d/dev-scripts.exports
```

Comment:
@derekhiggins This doesn't work on dualstack, and image-registry is broken there - you need both subnets